src/Console/Arguments.php

```diff
@@ -129,24 +129,44 @@ public function getArgument(string $name): ?string
     /**
      * Get an array of all the options
      *
-     * @return array<string, string|bool|null>
+     * @return array<string, list<string>|string|bool|null>
      */
     public function getOptions(): array
     {
         return $this->options;
     }
 
     /**
-     * Get an option's value or null
+     * Get a non-multiple option's value or null if not set.
      *
      * @param string $name The name of the option to check.
-     * @return string|bool|null The option value or null.
+     * @return string|bool|null
      */
     public function getOption(string $name): string|bool|null
     {
```

Comment:
I would add an assert or actual exception here. Will also make PHPStan happy then.

Reply:
That sounds good.